### DISCUSSION TITLE
Having separate "context dependent" tours

### DISCUSSION THREAD

**Discussion by GreenAsJade** [gilbarbara/react-joyride#824] [General]
The example shows us wrapping our App in <Joyride>, with a single set of states/steps/

What's the right way to do "multiple different tours" which started depending on what a user does?

I've seen a ticket from someone who tried to do something like this by dynamically instantiating joyride, but that seems a little "against the grain", and perhaps unsuprisingly bumping into issues.

Would hate to re-invent a wheel like this...

**Comment by gilbarbara** [comment]
hey @GreenAsJade 

I've used it that way, but I would render the Joyride using a `key` prop or maybe conditionally if that didn't work.
